src/core/core.cpp

```diff
@@ -141,10 +136,7 @@ ToxCorePtr Core::makeToxCore(const QByteArray& savedata, const ICoreSettings& se
 
     case TOX_ERR_NEW_LOAD_BAD_FORMAT:
         qCritical() << "Failed to parse Tox save data";
-        if (err != nullptr) {
-            *err = ToxCoreErrors::BAD_PROXY;
```

Comment:
Bonus bug fix. :P

Reply:
Yeah, no need to keep the wrong code :).